node/network/statement-distribution/src/vstaging/grid.rs

```diff
@@ -60,10 +60,10 @@ pub struct SessionTopologyView {
 impl SessionTopologyView {
 	/// Returns an iterator over all validator indices from the group who are allowed to
 	/// send us manifests.
-	pub fn iter_group_senders<'a>(
-		&'a self,
+	pub fn iter_group_senders(
+		&self,
 		group: GroupIndex,
-	) -> impl Iterator<Item = ValidatorIndex> + 'a {
+	) -> impl Iterator<Item = ValidatorIndex> + '_ {
```

Comment:
my grey beard is showing 😆

Reply:
Haha! I didn't even know this was a thing.